docs/components/dialogs/alert-dialog.md

```diff
@@ -0,0 +1,34 @@
+---
+sidebarDepth: 0
+---
+
+# AlertDialog
+
+This is an overview of the most common usage of AlertDialog.
+
+The `alert()` method shows a message and an OK button. Use it to show information and notifications that do not require an action from the user.
+
+## Live Demo
+<DocExampleBox codeBox="https://codesandbox.io/s/m8x5q5njx?module=%2Fsrc%2FApp.vue">
```

Comment:
you can remove the codebox link. also there is a prop for livedemos.
< DocExampleBox :liveDemoMode="true" >

Reply:
> you can remove the codebox link. also there is a prop for livedemos.
> < DocExampleBox :liveDemoMode="true" >

Fixed.